pkg/cmd/calico/calico_client.go

```diff
@@ -0,0 +1,63 @@
+package calico
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+	"github.com/dcos/dcos-core-cli/pkg/mesos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+)
+
+// Client is a calico client for DC/OS.
+type Client struct {
+	http *httpclient.Client
+}
+
+// NewClient creates a new calico client.
+func NewClient(baseClient *httpclient.Client) *Client {
+	return &Client{
+		http: baseClient,
+	}
+}
+
+// getCaCertificate gets the CA certificate from the server.
+func (c *Client) getCaCertificate() (string, error) {
```

Comment:
It's done here https://github.com/dcos/dcos-cli/blob/51eee402180b99bdf56ebe8356963b98807b8861/pkg/setup/setup.go#L131

Reply:
ca bundle path should be available in config `"core.ssl_verify"`

Reply:
But this is only getting done when there is a `unknown authority error`. Also for my cluster the `core.ssl_verify` field is empty.